src/main/java/org/zalando/nakadi/repository/TopicRepository.java

```diff
@@ -30,6 +48,9 @@
     Optional<PartitionStatistics> loadPartitionStatistics(Timeline timeline, String partition)
             throws ServiceUnavailableException;
 
+    List<Optional<PartitionStatistics>> loadPartitionStatistics(Collection<TimelineAndPartition> partitions)
```

Comment:
it's strange to see list of optionals. If the object is empty - it just should not be added to the list.